### DISCUSSION TITLE
AaC Starter Template Repository?

### DISCUSSION THREAD

**Discussion by Coffee2Bits** [DevOps-MBSE/AaC#184] [Archive]
In order to prep for demonstrating how an end user would pull down and run AaC I had to to setup a gitpod demo environment which is very close to what we'd want to use as a repo template for AaC users. 

Do we want to add couple minor tweaks and support a template repo for AaC users? I think it could be a great tool for users to get started immediately with minimal impact or effort.

The demo environment I used is the [demo-7-setup](https://github.com/jondavid-black/AaC/tree/demo-7-setup).

**Comment by jondavid-black** [comment]
Would the template repository be specific to GitHub or would it also work for GitLab?

**Reply by Coffee2Bits** [reply]
My original intent was Github, but we could certainly do more given that we're alright with the extra overhead.

**Comment by lizzcondrey** [comment]
This has been marked resolved, the template repo exists on GitHub, and is referenced in our documentation. Closing this discussion.
